components/scream/src/share/field/field_repository.hpp

```diff
@@ -412,6 +414,13 @@ registration_ends (const std::shared_ptr<const GridsManager>& gm) {
     }
   }
 
+  // Homme (and possibly other places in EAM) hard-codes water vapor as the 1st
+  // tracer in Q. Therfore, find qv in the TRACERS group, and make sure it's the 1st
+  auto qv_pos = ekat::find(tr_gr.m_fields_names,"qv");
```

Comment:
will this spit out an error if "qv" is not found at all?  On that same note, many of the unit tests probably won't have "qv", instead they'll have dummy variable names, but presumably we may create a test where dummy variable names are still treated like tracers.  

So long question short.  What will happen if we have some tracers and none of them are names "qv"?

Reply:
It shouldn't. This simply looks for it, and, if found, it makes sure `qv` is the 1st tracer. If not found, no big deal.